esmvaltool/config-references.yml

```
@@ -292,6 +297,10 @@ authors:
     name: Krasting, John
     institute: NOAA, USA
     orcid: https://orcid.org/0000-0002-4650-9844
+  kuehbacher_birgit:
+    name: Kuehbacher, Birgit
+    institute: DLR, Germany
+    email: birgit.kuehbacher@dlr.de
```

```suggestion
    email: birgit.kuehbacher@dlr.de
    orcid:
```

It's necessery to add an empty `orcid` entry here to render the HTML output file correctly.